.github/workflows/main.yml

```diff
@@ -105,7 +105,7 @@ jobs:
   linux-openj9:
     if: github.event.action != 'labeled'
     name: 'Linux (OpenJ9 8)'
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
@vlsi  I am going to close the PR, but maybe this correction should by applied to master? Would you take care of it?